spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/DocumentDbFactory.java

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
not necessary. core contains operation/context mapping context. and current folder organization is following spring data mongo, graph database, cassandra examples.